assets/queries/k8s/hpa_deployments_no_replicas/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "hpa_deployments_no_replicas",
+  "queryName": "HPA Deployments No Replicas",
+  "severity": "LOW",
+  "category": "Pod",
+  "descriptionText": "Confirm that the HPA doesn't have any static replicas in the deployment",
```

Comment:
A more suitable description would be "Deployments targeted by HorizontalPodAutoscaler should not have a statically configured replica count set"